lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -76,34 +67,28 @@ class DOMSize extends Audit {
       SCORING_MEDIAN
     );
 
-    const cards = [{
-      title: 'Total DOM Nodes',
-      value: Util.formatNumber(stats.totalDOMNodes),
-      target: `< ${Util.formatNumber(MAX_DOM_NODES)} nodes`,
-    }, {
-      title: 'DOM Depth',
-      value: Util.formatNumber(stats.depth.max),
-      snippet: depthSnippet,
-      target: `< ${Util.formatNumber(MAX_DOM_TREE_DEPTH)}`,
-    }, {
-      title: 'Maximum Children',
-      value: Util.formatNumber(stats.width.max),
-      snippet: widthSnippet,
-      target: `< ${Util.formatNumber(MAX_DOM_TREE_WIDTH)} nodes`,
-    }];
+    const headings = [
+      {key: 'totalNodes', itemType: 'text', text: 'Total DOM Nodes'},
+      {key: 'depth', itemType: 'text', text: 'DOM Depth'},
+      {key: 'width', itemType: 'text', text: 'Maximum Children'},
+    ];
+
+    const items = [
+      {
+        totalNodes: Util.formatNumber(stats.totalDOMNodes),
+        depth: `${Util.formatNumber(stats.depth.max)} (${depthSnippet})`,
```

Comment:
i dont know if this display will scale.

e.g: 

![image](https://user-images.githubusercontent.com/39191/37547738-b5deecfe-2930-11e8-8842-92e41ba57b98.png)

fwiw here is the same with the card:
![image](https://user-images.githubusercontent.com/39191/37547785-ea422f7e-2930-11e8-8b80-ee811ce8a2a6.png)

Reply:
~~heh good catch, do we really need the snippet displayed if it was just a tooltip before?~~

I like outer HTML 👍